apps/fluent-tester/src/TestComponents/Tooltip/TooltipTest.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import { TOOLTIP_TESTPAGE } from '../../../../E2E/src/Tooltip/consts';
+import { TooltipDefault } from './TooltipDefault';
+import { TooltipPosition } from './TooltipPosition';
+import { Test } from '../Test';
+import type { TestSection, PlatformStatus } from '../Test';
+
+const tooltipSections: TestSection[] = [
+  {
+    name: 'Default',
+    testID: TOOLTIP_TESTPAGE,
+    component: TooltipDefault,
+  },
+  {
+    name: 'Target + Positioning',
+    testID: TOOLTIP_TESTPAGE,
```

Comment:
>testID: TOOLTIP_TESTPAGE, [](http://example.com/codeflow?start=4&length=25)

Only the first test needs an ID. This is so that the e2e test is able to fine the element via the UIA tree.